config/locales/views/settings/fr.yml

```
@@ -154,8 +151,12 @@ fr:
           referential_html: Replace self-referential links with %{community}-specific links <p class='crayons-field__description'>If you check this box, the post will automatically change any URLs included in the post to refer to the version of that article on %{community} if available. This is primarily meant for folks migrating their entire blog onto %{community}.</p>
           save: Save Feed Settings
           submit: Submit Feed Settings
-        save_web: Save Web Monetization Settings
-        web: Web monetization
+        web_monetezation:
+          header: Web monetization
+          receive_payments: 'Receive micropayments for time spent reading your posts directly. '
```

Quick, non-blocking change: let's remove the extra space after the closing parenthesis here:
```suggestion
          receive_payments: 'Receive micropayments for time spent reading your posts directly.'
```